test/test_binomial.cpp

```diff
@@ -716,6 +716,19 @@ void test_spots(RealType T)
 
    check_out_of_range<boost::math::binomial_distribution<RealType> >(1, 1); // (All) valid constructor parameter values.
 
+  {
+     binomial_distribution<RealType> dist(9079765771874083840, 0.561815);
+#ifdef TEST_FLOAT
+     RealType expected = 5101149012694663168;
+#endif
+#ifdef TEST_DOUBLE
+     RealType expected = 5101148604445670400;
+#endif
+#ifdef TEST_LDOUBLE
+     RealType expected = 5101148604445669693;
+#endif
+     BOOST_CHECK_EQUAL(quantile(dist, 0.0365346), expected);
```

Comment:
I can reproduce the issue with a standalone executable built with ` g++ test.cc -I$PWD/include -std=c++17 -g -DBOOST_MATH_STANDALONE`.  However, when I try to put the test case into `test_binomial.cpp`, the original version just passed without stuck.  Is there any compiler flags that make it behave differently?

Reply:
I see it's `BOOST_MATH_DISCRETE_QUANTILE_POLICY`.  How can I test under the default value?

Reply:
The default is `#define BOOST_MATH_DISCRETE_QUANTILE_POLICY integer_round_outwards`, but at the top of this file it is defined as `real`. You can pass a different policy to the distribution as a template parameter.